### DISCUSSION TITLE
Add GraphiQL IDE for better experience

### DISCUSSION THREAD

**Discussion by DasMergo** [payloadcms/payload#2544] [Feature Requests & Ideas]
I think adding https://github.com/graphql/graphiql to Payload CMS Would be a game changer for DX
with schema and collection types support as they done it on GraphCms(Hygraph):
https://hygraph.com/images/docs/api-reference/basics/playground.png

**Comment by DanRibbens** [comment]
Are you asking for this? https://payloadcms.com/docs/graphql/overvie

Or something different?

**Reply by DasMergo** [reply]
cool, but GraphiQL is a Different IDE for graphql. so is more easier to work with mutations and filters also more documented playground. simply better GraphQL Playground.
and also with GraphiQL we can make advanced queries more fun and easier
this is online demo: https://cloud.hasura.io/public/graphiql
but u have to provide an endpoint
for example : https://rickandmortyapi.com/graphql

**Reply by kyuumeitai** [reply]
The graphiql explorer is so much better. What can be done to install it in a project?

**Reply by djerraballi** [reply]
Think the issue here is that graphql-playground has not been in support, has a lot of bugs and visually is unusable in its current state since hints get stuck on the screen. 

GraphiQL is still supported.

**Comment by tobidemski** [comment]
Are there any plans to update the graphql playground lib? I just started with payloadcms and the current one is very annoying when it comes to filling entries and the not disappearing labels. (Workaround: devtools --> delete the related div elements) I have seen the pr #8384 but its opened already for a long time.
Would really appreciate it if we could update this!
